test/stages/variables.tf

```diff
@@ -75,23 +69,30 @@ variable "gitops_namespace" {
 variable "git_username" {
 }
 
-variable "cluster_name" {
-  type        = string
-  description = "The name of the cluster"
-  default     = ""
-}
-
 variable "kubeseal_namespace" {
   default = "sealed-secrets"
 }
 
 variable "cp_entitlement_key" {
```

Comment:
Just want to be clear.... Where is the entitlement key being set?   We've gone back and forth on this conversation.  If it has not been set across the entire cluster, then it is needed in the namespace If it is needed in the namespace where this will be deployed, otherwise deployment will fail.   I don't see it getting set anywhere.   

We have a [separate issue](https://github.com/cloud-native-toolkit/software-everywhere/issues/23) to apply the entitlement key to the entire cluster.  In the meantime, we need to make sure that this automation can deploy successfully by including the entitlement key where it is needed.

Reply:
If the entitlement key is being set by a dependency, that is fine.  we just need to make sure we know where to find it.

Reply:
As discussed in office hours, entitilement key is not required, it is already set in cp4d instance